ninja-core/src/site/markdown/documentation/getting_started/installing_ninja.md

```diff
@@ -61,3 +61,19 @@ Make sure that you are using Maven 3 or greater
 If Maven is not available on your computer please 
 follow the following guide to setup Maven: http://maven.apache.org/guides/getting-started
 
+If you like to use the simplified form of calling the Ninja Maven plugin you have to
+define the pluginGroup within your <code>settings.xml</code>:
+
+<pre class="prettyprint">
+  .
+  <pluginGroups>
+    <pluginGroup>org.ninjaframework</pluginGroup>
+  </pluginGroups>
+  .
+</pre>
+
+otherwise you have to use the full qualified way of calling the Ninja Maven plugin via:
```

Comment:
Hmm... usually people include the Ninja plugin in their project's pom.xml. That way you can also simply call ninja:run... That is also what is recommended in the "first steps" section and simplifies things a lot (you don't have to learn about settings.xml, nor its location eg..)...

Do I miss something obvious?

It's pretty cool that you updated the documentation - but I nevertheless wanted to discuss the change :) No offence :)